css/src/components/stretched-link.scss

```diff
@@ -0,0 +1,11 @@
+.stretched-link {
+	&::before {
+		position: absolute;
+		content: '';
+		z-index: $zindex-stretched-link;
+		inset-inline-start: 0;
```

Comment:
I think these can be consolidated to `inset: 0`.

Reply:
Nice! Updated